unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldCommon/ToolTipController.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class ToolTipController : MonoBehaviour
+{
+    public float alphaSpeed = 3f;
+    public RectTransform tooltipRT;
+    public CanvasGroup tooltipCG;
+    public TextMeshProUGUI tooltipTxt;
+
+    Coroutine changeAlphaCor;
+
+    public void Desactivate()
+    {
+        if (changeAlphaCor != null)
+            StopCoroutine(changeAlphaCor);
+        changeAlphaCor = StartCoroutine(ChangeAlpha(1, 0));
+    }
+
+    public void OnHoverEnter(BaseEventData data)
+    {
+    
+        PointerEventData dataConverted = data as PointerEventData;
```

Comment:
```suggestion
        if (!(data is PointerEventData dataConverted)) 
                return;
```
Additionally adds a safe fallback if we dont receive a `PointerEventData` (unlikely but good practice)